common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -78,6 +78,10 @@ div.problem {
     display: inline;
   }
 
+  .exceeded-time-limit-message {
```

Comment:
@caesar2164 Is there a more general class/rule we can reuse here instead of special-casing this?